config.yaml

```diff
@@ -42,34 +42,18 @@ build_iso:
     packages_dir: "result/packages/latest"
```

Comment:
minor: It'd be good to mention somewhere in commit msg that you are not only *creating* new groups, but *replacing* the old packages groups.